src/TimeModeling/Types/ModelStructure.jl

```diff
@@ -14,9 +12,9 @@ export Model, PhysicalParameter, get_dt
 
 """
     PhysicalParameter
-        n::IntTuple
-        d::RealTuple
-        o::RealTuple
+        n::NTuple{N, T}
+        d::NTuple{N, Tf}
+        o::NTuple{N, Tf}
```

Comment:
Below is a generic "Tuple" in case someone wants to build the struct directly with weird values for some reason but this one is the types that will always be used by the constructor if called from Model